package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "rnw-front-end-development",
-  "version": "0.9.0",
+  "version": "0.9.1",
```

Comment:
I just merged something else in - please keep the version at 0.9.1 and add to the changelog, as we'll be doing one release with all the changes